pygit2/settings.py

```diff
@@ -101,4 +115,43 @@ def cache_object_limit(self, object_type, value):
         """
         return option(GIT_OPT_SET_CACHE_OBJECT_LIMIT, object_type, value)
 
+    @property
+    def ssl_cert_file(self):
+        """TLS certificate file path."""
+        raise AttributeError(
```

Comment:
@jdavid since we're tracking what we set anyway, we have all values to also expose this for reading. I think it'd be better consistency with this.

Reply:
Agree.